test/AWSUtils.Tests/StorageTests/PersistenceGrainTests_AWSDynamoDBStore.cs

```diff
@@ -47,22 +47,7 @@ protected override TestCluster CreateTestCluster()
                     options.ClusterConfiguration.Globals.RegisterStorageProvider<DynamoDBStorageProvider>("DDBStore1", new Dictionary<string, string> { { "DataConnectionString", dataConnectionString } });
```

Comment:
this tests are not included in vso or Jenkins. make sure you run it locally and it passes after this change.